src/js/dataMapping/financialAssistance/loanTransactionTable.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * loanTransactionTable.js
+ * Created by Emily Gullo 04/24/2017
+ */
+
+const tableSearchFields = {
+    columnWidths: {
```

Comment:
Please validate the column widths by temporarily disabling the custom font (fall back to default `sans-serif`) and testing on a variety of browsers to determine the smallest acceptable width for each. Add a bit of extra width for leeway.